src/main/java/de/bonndan/nivio/output/map/PNGRenderCache.java

```diff
@@ -93,23 +90,17 @@ public void onApplicationEvent(ProcessingFinishedEvent processingFinishedEvent)
     }
 
     private byte[] asByteArray(LandscapeImpl landscape) {
-        TranscoderInput input_svg_image = new TranscoderInput(new ByteArrayInputStream(getSVG(landscape).getBytes()));
-        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
-        TranscoderOutput transcoderOutput = new TranscoderOutput(outputStream);
+        TranscoderInput input_svg_image = new TranscoderInput(
+                new ByteArrayInputStream(getSVG(landscape).getBytes())
+        );
         Transcoder transcoder = new PNGTranscoder();
-        try {
+        try (ByteArrayOutputStream outputStream = new ByteArrayOutputStream()) {
```

Comment:
@BilligsterUser: nice that you fixed that error! Could you also write a test that fails with the old implementation and now passes?

Reply:
83ba1a4